docs/modules/ROOT/pages/cops_minitest.adoc

```diff
@@ -1509,7 +1514,7 @@ refute_instance_of(Class, object, 'message')
 | Yes
 | Yes
 | 0.10
-| -
+| 0.35.0
```

Comment:
No changes are required to this file.
https://github.com/rubocop/rubocop-minitest/blob/master/CHANGELOG.md?plain=1#L1-L10